converter/src/main/java/gov/cms/qpp/conversion/correlation/model/Correlation.java

```diff
@@ -0,0 +1,42 @@
+package gov.cms.qpp.conversion.correlation.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+	"correlationId",
+	"config"
+})
+public class Correlation {
+
+	@JsonProperty("correlationId")
```

Comment:
`@JsonProperty("correlationId")` is specified 3 times. I agree with @halprin that we do not need the `@JsonProperty....` code